src/main/java/de/chojo/repbot/data/wrapper/ReputationLogEntry.java

```diff
@@ -70,4 +70,9 @@ public ThankType type() {
     public LocalDateTime received() {
         return received;
     }
+
+    public String timestamp(){
+        var timestamp = ((messageId() >> 22) + 1420070400000L) / 1000;
```

Comment:
I would suggest moving this magic number to a constant.
Also we can just divide the discord epoch itself by 1000 in the constant and remove the division here

Reply:
We need to divide the bitshifted snowflake by 1000 anyway. I dont think this will have any improvement if we precompute the epoch divided by 1000. But we can move the epoch to a constant.

Reply:
Oh yes, I have forgotten that we need to divide the snowflake itself also by 1000. So just moving the epoch to a constant is fine